TestApp/app/screens/DataScreen.js

```diff
@@ -103,10 +103,10 @@ export default class DataScreen extends Component {
               this.setState({ docTtl: itemValue })
             }
           >
-            <Picker.Item label="Default" value="-1" />
-            <Picker.Item label="No Cache" value="0" />
-            <Picker.Item label="2 seconds" value="2" />
-            <Picker.Item label="Infinite" value="-1" />
+            <Picker.Item label="Default" value={-1} />
```

Comment:
Using a hard-coded value kind of defeat the purpose of testing different TTL values.

Can we use `Data.TimeToLive.NO_CACHE` as in https://github.com/microsoft/appcenter-sdk-react-native/blob/develop/appcenter-data/Data.d.ts#L4-L8?